db/sql/test_sql_functions.sql

```diff
@@ -1995,11 +1998,11 @@ DECLARE
   ]$j$;
 BEGIN
   PERFORM __setup_column_alter();
-  RETURN NEXT is(msar.alter_columns('col_alters'::regclass::oid, col_alters_jsonb), ARRAY[2, 3, 4]);
-  RETURN NEXT col_type_is('col_alters', 'col1', 'character varying(48)');
-  RETURN NEXT col_type_is('col_alters', 'col2', 'integer');
-  RETURN NEXT col_default_is('col_alters', 'col2', 5);
-  RETURN NEXT col_type_is('col_alters', 'Col sp', 'integer');
+  RETURN NEXT is(msar.alter_columns('test_schema.col_alters'::regclass::oid, col_alters_jsonb), ARRAY[2, 3, 4]);
+  RETURN NEXT col_type_is('test_schema', 'col_alters', 'col1', 'character varying(48)', 'type should be varchar');
+  RETURN NEXT col_type_is('test_schema', 'col_alters', 'col2', 'integer', 'type should be integer');
```

Comment:
I had to include descriptions for certain tests so that the functions resolve to the correct overload.